about.php

```diff
@@ -1,7 +1,12 @@
+<?php
+require_once("php/git.php")
```

Comment:
While it can be used as one, it's not actually a function - https://www.php.net/manual/en/function.require-once.php

And should have a trailing `;` too.

`require_once "php/git.php";`

Reply:
I really appreciate these comments - I'm new to PHP. Will fix.

Reply:
Doesn't look like this one got fixed... Or got lost :)